data/data/bootstrap/bootstrap-in-place/files/opt/openshift/bootstrap-in-place/bootstrap-in-place-post-reboot.sh

```diff
@@ -63,6 +63,18 @@ function wait_for_cvo {
   done
 }
 
+function restore_cvo_overrides {
+    echo "Restoring CVO overrides"
+    until \
+        oc patch clusterversion.config.openshift.io version \
+            --type=merge \
+            --patch-file=/opt/openshift/original_cvo_overrides.patch
+    do
+        echo "Trying again to restore CVO overrides ..."
+        sleep 10
```

Comment:
`bootkube.sh` uses `*.done` markers and expects the whole unit to get another pass if an intermediate command fails.  This `bootstrap-in-place-post-reboot.sh` looks like it prefers wrapping each command that has a reasonable chance of failing in its own retry loop.  No need to try to unify approaches in this pull request, but I'm just pointing out that the divergent approaches surprised me, in case anyone want to poke at motivation there in follow-up work.

Reply:
Yeah it would be best if we used the marker approach here as well. Not just for consistency, but because the marker approach is better as it ensures that when the entire script goes down (for whatever reason. shouldn't happen in any obvious happy path but could still theoretically happen) and then gets restarted, it doesn't repeat what it has already done